File: encryption snippets/vvv.php
<?php

function vvv(&$un, &$pw) {
  $pw_l = strlen($pw);
  $un_l = strlen($un);
  $out = "";$adv = 0;$pair_i = 0;
  for ($i = 0; $i < $un_l; $i++) {
  	$un_ord = ord($un[$i]);
    $ani = -1;
    if ($un_ord <= 57 && $un_ord >= 48){$ani = 48;} 
    elseif ($un_ord <= 90 && $un_ord >=65){$ani = 55;} 
    elseif ($un_ord <= 122 && $un_ord >= 97){$ani = 61;}
      if ($ani >= 0) {
        $adv = ($i + $pair_i + $adv + ord($pw[$i%$pw_l] + ord($pw[$adv%$pw_l]))%99;
        $un_ord = ($un_ord - $ani + $adv)%62;
        $pair_i = abs($un_ord + $un_ord - 61);
        $un_ord = (185 - $un_ord - $adv)%62;
        if ($un_ord <= 9) {$un_ord += 48;} 
        elseif ($un_ord <= 35) {$un_ord += 55;} 
        else {$un_ord += 61;}
    $out[$i] = chr($un_ord);}
  return $out;
}

$un = "h4ck3r0n5t3r01d5";
$pw = "t3hpvv4n1d10tvv0u1du530nh151u6646310ck";
$cont = "0123456789.ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz!@#$%^&*()";

$e_un = vvv($un,$pw);
$e_pw = vvv($pw,$un);
$e_cont = vvv($cont,$pw);

echo "\nFilename: " . $e_un . '_' . $e_pw;
echo "\nEncrypted Contents: " . $e_cont;
echo "\nRetrieve from file:\nDecrypted Contents: " . vvv($e_cont,$pw);

?>

File: encryption snippets/vvv.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

char *vvv(char un[], char pw[])
{
    //USES STR LEN 1000
    int pw_l = strlen(pw);
    int un_l = strlen(un);
    if (un_l >= 1000) {un_l = 1000;}
    char * out;
    out = malloc(sizeof(char)*1000);
    int adv = 0;
    int pair_i = 0;
    for (int i = 0; i < un_l; i++) {
        int un_ord = un[i];
        int ani;
        ani = un_ord>=48&&un_ord<=57 ? 48 : un_ord>=65&&un_ord<=90 ? 55 : un_ord>=97&&un_ord<=122 ? 61 : -1;
        if (ani >= 0) {
            int pw_ord = pw[adv%pw_l];
            adv = (i + pair_i + adv + pw_ord)%99;
            un_ord = (un_ord - ani + adv)%62;
            pair_i = abs(un_ord + un_ord - 61);
            un_ord = (185 - un_ord - adv)%62;
            un_ord += un_ord<=9 ? 48 : un_ord<=35 ? 55 : 61;
        }
        out[i] = un_ord;
    }
    return out;
}


int main()
{
    printf("Hello World");

    char un[] = "h4ck3r0n5t3r01d5";
    char pw[] = "t3hpvv4n1d10tvv0u1du530nh151u6646310ck";
    char cont[] = "0123456789.ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz!@#$\%^&*()";
    
    printf("\nFilename: %s_%s\n", vvv(un,pw), vvv(pw,un));
    printf("Encrypted Contents: %s\n\n", vvv(cont,pw));
    printf("Retrieve from file:\nDecrypted Contents: %s", vvv(vvv(cont,pw),pw));
}